packages/core/directives/map.ts

```diff
@@ -358,6 +358,11 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
    */
   @Output() mapReady: EventEmitter<any> = new EventEmitter<any>();
 
+  /**
+   * This event is fired when the visible tiles have finished loading.
+   */
+  @Output() tilesLoaded: EventEmitter<void> = new EventEmitter<any>();
```

Comment:
you still left `any` in the initializer part

Reply:
You're right, fixed!